app/controllers/deadlines_controller.rb

```diff
@@ -0,0 +1,37 @@
+class DeadlinesController < ApplicationController
+  before_action :authenticate_user!
+
+  # POST deadlines/toggle
+  def toggle
+    current_user.update_attributes!(mode_params)
+    redirect_to :back, notice: "Account details updated successfully. We'll remember that."
+  end
+
+  # POST deadlines
+  def create
+    Deadline.create!(deadline_params.merge(user: current_user))
+    notice = "Locked and loaded: this lesson is going down by " +
+      "#{deadline_params[:target_completed_on]}. You'll receive a reminder for " +
```

Comment:
Line is too long. [82/80]<br>Align the operands of an expression in an assignment spanning multiple lines.